src/pages/test-report/components/toolbar/toolbar.ts

```diff
@@ -23,13 +24,18 @@ export class ToolbarComponent {
   componenetState: ToolbarComponentState;
   subscription: Subscription;
 
+  isRemoveFaultMode: boolean = false;
   isSeriousMode: boolean = false;
   isDangerousMode: boolean = false;
 
-  constructor(private store$: Store<StoreModel>) {}
+  constructor(private store$: Store<StoreModel>) { }
 
   ngOnInit(): void {
     this.componenetState = {
```

Comment:
I know it's just a typo and pre-existed but componenetState should be componentState